db/migrate/20250225000050_remove_contents_from_issue_contents.rb

```diff
@@ -0,0 +1,74 @@
+# For auto load
+FullTextSearch::Migration
+
+class RemoveContentsFromIssueContents < ActiveRecord::Migration[5.2]
+  def up
+    return if !table_exists?(:issue_contents)
+
+    if Redmine::Database.mysql?
+      remove_index :issue_contents, :contents
+    else
+      remove_index :issue_contents, name: "index_issue_contents_pgroonga"
+    end
+    remove_column :issue_contents, :contents, :text
+    remove_column :issue_contents, :is_private, :boolean
+    contents_limit = Redmine::Database.mysql? ? 16.megabytes : nil
+    add_column :issue_contents, :content, :text, limit: contents_limit
+
+    if Redmine::Database.mysql?
+      add_index :issue_contents,
+                :content,
+                type: "fulltext",
+                comment: "TOKENIZER 'TokenMecab'"
+    else
+      add_index :issue_contents,
+                [:id,
+                 :project_id,
+                 :issue_id,
+                 :subject,
+                 :content,
+                 :status_id],
+                name: "index_issue_contents_pgroonga",
+                using: "PGroonga",
+                with: [
+                  "tokenizer = 'TokenMecab'",
+                  "normalizer = 'NormalizerNFKC121'",
```

Comment:
Could you give me comments about which normalizers and tokenizer we should use?
Or we don't have to specify the tokenizer here like the other search features.

Reply:
We should use multilingual morphological based tokenizer but we haven't implemented it yet. See also: https://github.com/groonga/groonga/issues/1941

So we can use `TokenMecab` for now but we should implement multilingual morphological based tokenizer as fast as possible and replace this.

We should use the latest NFKC based normalizer. Can we detect the best normalizer based on the runtime Groonga version? For example, we want to use `NormalizerNFKC` with Groonga 14.1.3 or later.

FYI:
https://github.com/clear-code/redmine_full_text_search/blob/513720b9f1bb93ded1db42dc4b1075bbaa76ed82/lib/full_text_search/mroonga.rb#L47-L51

https://github.com/clear-code/redmine_full_text_search/blob/513720b9f1bb93ded1db42dc4b1075bbaa76ed82/lib/full_text_search/pgroonga.rb#L33-L41

Reply:
fix: [0686687](https://github.com/clear-code/redmine_full_text_search/pull/165/commits/068668702421535f4969dc532fcbe709200fce1c)
Thank you for giving me the advice. After implementing this feature, I will start to do https://github.com/groonga/groonga/issues/1941 👍🏾